### ISSUE TITLE
Crashing while ream get default instance.

### ISSUE THREAD

**Issue by stories2** [I2MAX-LearningProject/ClientApp#1]
## Problem File Name
RealmController.java
## Expected behavior
Getting default instance from realm.
## Actual behavior
It goes crash.
## Steps to reproduce the behavior
- Open this project.
- Run.
## Program printed log
```
08-09 15:24:16.044 21191-21191/com.machine.i2max.i2max I/i2max: [I2maxMain] {Init} (preparing ui)
08-09 15:24:16.120 21191-21191/com.machine.i2max.i2max I/i2max: [RealmController] {RealmController} (Init realm)
08-09 15:24:16.129 21191-21191/com.machine.i2max.i2max E/i2max: [RealmController] {RealmController} (Error: The 'SellingDataTable' class is missing from the schema for this Realm.)
```

**Comment by stories2** [comment]
We asked question at [stackoverflow](https://stackoverflow.com/questions/45583336/the-class-is-missing-from-the-schema-for-this-realm/45583438#45583438)

**Comment by stories2** [comment]
We fixed at e56e281
